### ISSUE TITLE
exporting string literal type fires no-unused-expression rule

### ISSUE THREAD

**Issue by zam6ak** [palantir/tslint#1132]
### Bug Report

- __TSLint version__: `3.6.0`
- __TypeScript version__: `1.8.9`
- __Running TSLint via__: gulp-tslint v3.6.0

#### TypeScript code being linted

```ts
// string literal type
export type MyFlag = 'One' | 'Two';
// ----[14, 28]------^
export function processSomething(flag: MyFlag): string { 
return 'ommited for brevity...';
}
```

with `tslint.json`:

```json
"no-unused-expression": true,
```

#### Actual behavior
```[gulp-tslint] error (no-unused-expression) app.ts[14, 28]: expected an assignment or function call```

#### Expected behavior

no error

**Comment by JKillian** [comment]
@zam6ak I couldn't duplicate this. Maybe there's an old version of TSLint or TypeScript in your linting pipeline somewhere?

**Comment by zam6ak** [comment]
@JKillian  Hmm...You are probably right...I cloned the repo to another box and it worked...Closing it!

**Comment by JKillian** [comment]
Glad it's working, cheers!
